Makefile

```diff
@@ -112,6 +120,16 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | ${CLUSTER_CLI} delete -f -
 
+# Build the operator image
+operator-build:
+	BUILDAH_FORMAT=docker ${CONTAINER_CLI} build . -t ${IMG}
+
+# Push the operator image
+operator-push:
+	${CONTAINER_CLI} push ${IMG}
+
+OS    = $(shell uname -s | tr '[:upper:]' '[:lower:]')
+ARCH  = $(shell uname -m | sed 's/x86_64/amd64/')
```

Comment:
Let's move these to the top with the rest of the var definition

Reply:
Moded those, thanks